src/components/theme/View/RenderBlocks.jsx

```diff
@@ -23,14 +25,15 @@ const messages = defineMessages({
 });
 
 const RenderBlocks = (props) => {
-  const { content, intl, location, metadata } = props;
+  const { content, location, metadata, blockWrapperTag } = props;
+  const intl = useIntl();
   const blocksFieldname = getBlocksFieldname(content);
   const blocksLayoutFieldname = getBlocksLayoutFieldname(content);
   const blocksConfig = props.blocksConfig || config.blocks.blocksConfig;
   const CustomTag = props.as || React.Fragment;
 
   return hasBlocksData(content) ? (
-    <CustomTag>
+    <MaybeWrap condition={CustomTag} as={CustomTag}>
```

Comment:
How would CustomTag not evaluate to true here, given that it is defaulted to React.Fragment on line 33?

Reply:
Fixed.